README.md

```diff
@@ -1,23 +1,15 @@
 [![Periodic](https://github.com/nuclio/nuclio/actions/workflows/periodic.yaml/badge.svg?branch=development)](https://github.com/nuclio/nuclio/actions/workflows/periodic.yaml)
+[![Documentation Status](https://readthedocs.org/projects/nuclio/badge/?version=latest)](https://nuclio.readthedocs.io/en/latest/?badge=latest)
```

Comment:
As per Gilad's request, we will change the default docs version to `stable`.
But currently stable builds fail, I guess because we need to release the changes first (master branch doesn't have the readthedocs config yet).

Reply:
then we will change back to stable once released